uefi-mkconfig

```diff
@@ -53,156 +25,86 @@ add_uefi_entries () (
 		local efi_file_path
 		efi_file_path="${efi_file//$partition_mount}"
 
-		# Add entry if it doesn't exist
-		if ! check_if_uefi_entry_exists; then
-		
-			# Find first free hex address larger than or equal to 0100 for new UEFI boot entry
-			local bootnum
-			bootnum=256 # 256 is decimal value of 0100
-			while [[ "$(efibootmgr -u)" == *"Boot$(printf %04X $bootnum)"* ]]; do
-				bootnum=$((bootnum + 1))
-				
-				# Die if script exceeds managed range 0100-0200
-				if [[ $bootnum -gt 512 ]]; then
-					die "All IDs, within managed range by uefi-mkconfig, are full!"
-				fi
-
-			done
+		# Find first free hex address larger than or equal to 0100 for new UEFI boot entry
+		local bootnum
+		bootnum=256 # 256 is decimal value of 0100
+		while [[ "$(efibootmgr -u)" == *"Boot$(printf %04X $bootnum)"* ]]; do
+			bootnum=$((bootnum + 1))
```

Comment:
And then the first available one will also always be 0100

Reply:
It will be 0100 only for the first entry.

I suppose we don't need to check for existence every one of them and just use simple counting in a variable. But this is more reliable. And againm the code is already written so why not leave it there.